pilot/cmd/pilot-agent/main.go

```diff
@@ -582,17 +609,17 @@ func waitForCerts(fname string, maxWait time.Duration) {
 	for {
 		_, err := os.Stat(fname)
 		if err == nil {
-			return
+			return true
 		}
 		if !os.IsNotExist(err) { // another error (e.g., permission) - likely no point in waiting longer
 			log.Errora("error while waiting for certificates", err.Error())
-			return
+			return false
 		}
 
 		now := time.Now()
 		if now.After(endWait) {
 			log.Warna("certificates still not available after", maxWait)
```

Comment:
function name is wait for file now but this mentions certs, maybe a bit confusing now